pkgs/tools/misc/lokalise2-cli/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "lokalise2-cli";
+  version = "2.6.0";
+
+  goPackagePath = "github.com/lokalise/lokalise-cli-2-go";
```

Comment:
this doesn't get used with `buildGoModule`

Reply:
Thank you for your review @jonringer. I've amended the commit your feedback.